static/js/utils.js

```diff
@@ -41,7 +41,7 @@ const propertyName = (property) => {
 const propertyDescription = (property) => {
     if (columnsReady) {
         try {
-            return window.columns["keys"][property]["description"]
+            return window.columns.filter(c => c["key"] === property)[0]["description"]
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7256353)
```suggestion
            return window.columns.filter((c) => c["key"] === property)[0]["description"]
```